styles/stylesheet.css

```diff
@@ -0,0 +1,521 @@
+/* Body and general styling */
+body {
+  background-color: #ffffff;
+  color: #000000;
+  padding: 0;
+  border: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+h3 {
+  color: #3b6860;
+  font: 2rem "Irish Grover", cursive;
+}
+.hidden {
+  display: none;
+}
+h4 {
+  color: #3b6860;
+  font: 1.6rem "Irish Grover", cursive;
+}
+h5 {
+  color: #3b6860;
+  font: 1.3rem "Irish Grover", cursive;
+}
+p {
+  font: 1rem/1.5rem Arial, Helvetica, sans-serif;
+}
+a:link, a:visited {
+  text-decoration: none;
+  font: 1rem Arial, Helvetica, sans-serif;
+  font-weight: bold;
+  color: #274841;
+}
+a:hover, a:active {
+  color: #ffffad;
+  text-shadow: .2rem .2rem .3rem #274841;
+}
+button, a.button {
+  display: inline-block;
+  padding: .3rem .4rem;
+  border: solid .2rem #3b6860;
+  border-radius: .2rem;
+  color: #274841;
+  background-color: #85bbb1;
+  text-align: center;
+  font-variant: small-caps;
+  font-weight: bold;
+  line-height: 1rem;
+  box-sizing: content-box;
+  height: 1rem;
+}
+a.button:hover, a.button:active {
+  color: #ffffad;
+  text-shadow: .2rem .2rem .3rem #274841;
+}
+button:hover, button:active {
+  color: #ffffad;
+  text-shadow: .2rem .2rem .3rem #274841;
+}
+a:hover h4, a:active h4 {
+  color: #ffffad;
+  text-shadow: .2rem .2rem .3rem #274841;
+}
+
+/* Header styling */
+
+header {
+  width: 100%;
+  height: auto;
+  background: url("../images/header-image.jpg");
```

Comment:
Great job with the image. Remember to include the gradient per assignment requirement. This is so to act as a backup in case the image doesn't load.